buildmaster/LHCB_DY_13TEV_DIELECTRON/metadata.yaml

```diff
@@ -0,0 +1,53 @@
+# Previously called `LHCB_Z_13TEV_DIELECTRON`
+# author: "Tanjona R. Rabemananjara"
```

Comment:
Please don't add authorship, git keeps track of that (and more accurately since the data might change with time hence the versioning).

The "previously called" might make more sense as a "version comment" (but in any case not mandatory as it will be in `dataset_names` as well)